app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None | bool:
+    command = command.split(" ")[1:]
+    if command[0] != command[1]:
+        with (open(command[0], "r") as file_in,
+              open(command[1], "w") as file_out):
+            file_out.write(file_in.read())
+        if open("file.txt").read() == open("new_file.txt").read():
+            return True
+    return
```

Comment:
You can `call retrun` at the start of your func , and just check if file_name 1 == file_name 2 and then just write the file into another